pyproject.toml

```
@@ -0,0 +1,47 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = [
+  "hatch-vcs",
```

```suggestion
  "hatch-vcs",
  "hatchling",
```
I believe you should explicitly add the build system itself as a dependency (though I think the reason this worked is because `hatchling` might be a dependency of `hatch-vcs`).